Behavioral.Automation.Template/readme.md

```diff
@@ -0,0 +1,15 @@
+# Quantori Behavioral Automation Testing System
+Copyright (c) 2022 Quantori.
+
+Quantori Behavioral Automation is an open-source framework for UI testing automation based on Selenium and Specflow which incorporates good portability and ability to be integrated into a custom web-application. Quantori Behavioral Automation is designed to be used by both manual and automation QA engineers in projects which are built based on Behaviour Driven Development approach.
+
+## KEY FEATURES
+* Allows to create project for BDD Automation needs using Behavioral.Automation framework
+
+## Build instructions
+Using cmd run these commands in order to create Behavioral.Automation project from template
+dotnet new --install C:\Temp\Behavioral.Automation.Template.0.0.2.nupkg from command line --TODO replace with link after publishing the nuget
```

Comment:
`--TODO` ? :)

Reply:
Yeah :D I don't know how it'll exactly look like when we publish the nuget

Reply:
@vladimir-s-quantori Discussed with @andrey-brovko , I cleaned this section of readme and just left 'TBD' text, will fill after we deploy template to nuget.org